main.py

```diff
@@ -1,10 +1,37 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_json:
+        players = json.load(players_json)
+    for player, data in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=data["race"]["name"],
+            description=data["race"]["description"]
+        )
+
+        for skill in data["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race
+            )
+
+        guild, _ = Guild.objects.get_or_create(
+            name=data["guild"]["name"],
+            description=data["guild"]["description"]
+        ) if data["guild"] else (None, _)
```

Comment:
Why do you need  the underscore `_`

Reply:
When using get_or_create we get a tuple back, also If the player is not a member of a guild we also need to return it to a tuple,
If delete underscore we get error.